### ISSUE TITLE
"fn deploy --all" rolls back new-style func.yaml files to old ones

### ISSUE THREAD

**Issue by denismakogon** [fnproject/cli#412]
Before `fn deploy --app videbox`:
```
schema_version: 20180708
name: bucket-cleaner
version: 0.0.7
runtime: go
entrypoint: ./func
format: json
type: async
memory: 258
timeout: 360
triggers:
- name: bucket-cleaner
  type: http
  source: /bucket-cleaner
```

After:
```yaml
name: bucket-cleaner
version: 0.0.8
runtime: go
entrypoint: ./func
type: async
memory: 258
format: json
timeout: 360
```

Deploy log from one of the functions:
```
 fn --verbose deploy --all --local
Deploying bucket-cleaner to app: videobox at path: /bucket-cleaner
Bumped to version 0.0.8
Building image denismakogon/bucket-cleaner:0.0.8 
FN_REGISTRY:  denismakogon
Current Context:  default
Sending build context to Docker daemon  88.75MB
Step 1/8 : FROM fnproject/go:dev as build-stage
 ---> fac877f7d14d
Step 2/8 : WORKDIR /function
 ---> Using cache
 ---> 1980b829219e
Step 3/8 : ADD . /go/src/func/
 ---> da53a9713f79
Step 4/8 : RUN cd /go/src/func/ && go build -o func
 ---> Running in 9782a58d5d0c
Removing intermediate container 9782a58d5d0c
 ---> 99693a5edc9b
Step 5/8 : FROM fnproject/go
 ---> 76aed4489768
Step 6/8 : WORKDIR /function
 ---> Using cache
 ---> 63179d25b899
Step 7/8 : COPY --from=build-stage /go/src/func/func /function/
 ---> Using cache
 ---> 517ee1e48507
Step 8/8 : ENTRYPOINT ["./func"]
 ---> Using cache
 ---> 8c535700ceff
Successfully built 8c535700ceff
Successfully tagged denismakogon/bucket-cleaner:0.0.8

Updating route /bucket-cleaner using image denismakogon/bucket-cleaner:0.0.8...
```

As you may see `fn deploy --all` reverts func.yaml to what it was before (before we defined migration versions).

**Comment by JadeRedworth** [comment]
@denismakogon [PR 404](https://github.com/fnproject/cli/pull/404) will fix this issue.

**Comment by denismakogon** [comment]
How soon will this be merged? @JadeRedworth

**Comment by rikgibson** [comment]
Duplicate of #395

**Comment by JadeRedworth** [comment]
Issue resolved by: [#404 Removes routes and migrates everything to V2 APIs](https://github.com/fnproject/cli/pull/404)
